lib/classes/Variables.js

```diff
@@ -344,11 +344,24 @@ class Variables {
     }, valueToPopulate);
   }
 
-  warnIfNotFound(variableString, valueToPopulate) {
+  deleteNullEnvironmentVariables() {
```

Comment:
Yeah, tbh, I did have second thoughts about putting this here when I was implementing it - it does seem like a different concern.

I'll take a stab at moving it to the `package` or `deploy` plugins.